src/gui_common/GUICommon.cs

```diff
@@ -48,20 +43,9 @@ private GUICommon()
     public bool IsAnyExclusivePopupActive => GetCurrentlyActiveExclusivePopup() != null;
 
     /// <summary>
-    ///   The audio player for UI sound effects.
-    /// </summary>
-    private AudioStreamPlayer AudioSource { get; }
-
-    /// <summary>
-    ///   Second audio player for GUI effects. This is used if the primary one is still playing the previous effect.
+    ///  The audio players for UI sound effects.
```

Comment:
Is this missing one space (so there's 2 instead of 3) on this line?

Reply:
Yes, fixed it.